README.md

```diff
@@ -82,9 +82,21 @@ Explain language, locations key.
 
 ### config.rb
 
+To configure CuttingEdge specific settings in `config.rb`, you can run `cutting_edge` with the `--config` switch (you can also specify an alternative location for the config file). Always make sure you are defining your settings from within the `CuttingEdge` module. For instance:
+
+```ruby
+module CuttingEdge
+  REFRESH_SCHEDULE = '2h'
+end
+```
+
+Our sample [config.rb](config.rb) contains (within its comments) some examples of constants that you may configure. Here are some descriptions of what the less obvious ones achieve:
```

Comment:
The sample config (instead of Our).

that you may configure -> that you may wish to